### DISCUSSION TITLE
Issue with installing mitsuba-blender

### DISCUSSION THREAD

**Discussion by skille97** [mitsuba-renderer/mitsuba-blender#92] [Q&A]
Every time I try to install mitsuba blender I get the following error.
`RuntimeError: Cannot activate mitsuba-blender add-on. Python pip module cannot be initialized.
`
I use blender 4.0.2 on Arch Linux. 
Thanks for the help in advance.

**Comment by bathal1** [comment]
Can you please specify the following things:
- How did you install blender ?
- What steps are you taking to install the add-on ?

**Reply by skille97** [reply]
I have installed blender through pacman. 

I have followed the instructions giving in [Installation & Update Guide
](https://github.com/mitsuba-renderer/mitsuba-blender/wiki/Installation-&-Update-Guide)

**Reply by cnrgl** [reply]
i have not use custom path option as written in instructions; instead i started blender from console with "--python-use-system-env" option may be point same folder works for it
